docs/dsl/routes.md

```diff
@@ -195,34 +195,18 @@ trait Routes[-Env, +Err] {
 }
 ```
 
-## Converting `Routes` to `HttpApp`
-
```

Comment:
Instead of removing this section, I think we can refactor it and move it to the serving section to explain why we need handled routes to serve them.